source/configure/plugins-configuration-settings.rst

```diff
@@ -406,6 +406,38 @@ The URL to a running `rtcd <https://github.com/mattermost/rtcd>`__ service insta
 
 This is an optional field. Changing this setting requires a plugin restart to take effect.
 
+Enable call recordings (beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-selfhosted-only.rst
+  :start-after: :nosearch:
+
+**True**: Allow call hosts to record meeting video and audio on the cloud. Recordings include the entire call window view along with participants' audio track and any shared screen video.
```

Comment:
What's the default value for this config setting?

Reply:
Good question. I think it's **false**. @streamer45, can you confirm?

Reply:
Default would be **false** since the feature requires extra setup in order to work.